pandas/plotting/_matplotlib/core.py

```diff
@@ -102,13 +104,15 @@ def __init__(
         table=False,
         layout=None,
         include_bool=False,
+        column=None,
         **kwds,
     ):
 
         import matplotlib.pyplot as plt
 
         self.data = data
         self.by = by
+        self.column = [column] if not isinstance(column, list) else column
```

Comment:
Minor but I think this should be called `self.columns` since we are dealing with a collection

Reply:
i agree, but here the reason i chose `column` was for consistency reason, now it is aligned with argument name we already have `df.hist()` where it uses `column`

Reply:
I agree with @WillAyd here. I see your point. But `column` can be a single column or `None`. This is always a list, so `self.columns` is better.

When `column` is `None`, this will be `[None]`, while I guess should be `[]`. Can you have a look and see if anything should be changed?